lib/utils/args.js

```diff
@@ -54,21 +54,24 @@ function _normaliseFlagsAndArs(argsIntoCommand, cmd) {
     // bail on old style commands
     return argsIntoCommand;
   }
-  var processArgs = _.clone(process.argv);
+
+  const normalisedArgs = _.clone(argsIntoCommand);
```

Comment:
For the cases where more than one parameter is required, it doesn't work well expected for me. Could you check the tests performed and tell me your position?